src/modules/agora/AgoraClient.ts

```diff
@@ -65,7 +65,7 @@ export class AgoraClient implements FullNodeAPI
         this.client = axios.create({
             baseURL: endpoint.toString(),
             // Timeout are in ms, so 2s timeout
```

Comment:
Since we may change the value again, I think we can remove.
```suggestion
            // Timeout is in ms
```

Reply:
Good eye! Thanks.